e2e-tests/e2e-fixtures/mock-gef-facilities.js

```diff
@@ -227,9 +227,19 @@ const anUnissuedCashFacilityWith20MonthsOfCover = ({ facilityEndDateEnabled = fa
   canResubmitIssuedFacilities: null,
 });
 
+/**
+ * @param {{ facilityEndDateEnabled?: boolean}} options
+ * @returns {import('@ukef/dtfs2-common').Facility }
+ */
+const anIssuedContingentFacility = ({ facilityEndDateEnabled = false } = {}) => ({
+  ...anUnissuedContingentFacility({ facilityEndDateEnabled }),
+  hasBeenIssued: true,
+});
+
 exports.multipleMockGefFacilities = ({ facilityEndDateEnabled = false } = {}) => ({
   unissuedCashFacility: anUnissuedCashFacility({ facilityEndDateEnabled }),
   issuedCashFacility: anIssuedCashFacility({ facilityEndDateEnabled }),
   unissuedContingentFacility: anUnissuedContingentFacility({ facilityEndDateEnabled }),
   unissuedCashFacilityWith20MonthsOfCover: anUnissuedCashFacilityWith20MonthsOfCover({ facilityEndDateEnabled }),
+  anIssuedContingentFacility: anIssuedContingentFacility({ facilityEndDateEnabled }),
```

Comment:
Can we call this `issuedContingentFacility` to be inline with the other names please e.g `unissuedContingentFacility`

Reply:
Good spot, have changed